### PR TITLE
June update

### PR DESCRIPTION
**Pull request by cozybun**

### CODE REVIEW

File: draft.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Blockchain technology has three main challenges, for which we provide three solu
 
 **Privacy**. A trustless environment and anonymity are similarly essential to the wider adoption and use of the blockchain. We have a natural desire to keep our finances and transactions private. Emotiq is inspired by Monero and ZeroCash to provide additional privacy solutions, for example, by keeping transaction amounts hidden.
 
-**Usability**. Smart contracts are inaccessible for the majority of users. They’re difficult to use, hard to read, and impossible to write without the risk of introducing new security holes (as shown by the widely publicized hacks of Ethereum). It is impossible to understand the behavior of current smart contracts without using a computer to evaluate them. Legal contracts, in contrast, can be easily read and understood.
```
Comment (joelreymont) [inline draft.md:23]: Please leave usability in place.

File: draft.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ A natural language approach to smart contracts will attract the millions of indi
 
 ## 4.2 Oracles
 
-Smart contract creators place oracles as pre-built components into their ‘Lego box’. Developers are incentivized to create new oracles through a pay-per-transaction model (i.e., each time an oracle provides an answer, the developer of the oracle is paid). We aim to gather a library of oracle components to satisfy an ever-broadening scope for smart contracts, similar to how the Unity 3D Asset Store provides a database of 3D characters for game developers.
```
Comment (joelreymont) [inline draft.md:114]: Please leve the single quotes in place everywhere. These may be important when processing the whitepaper. Please try to focus on substantial changes and limit the tweaking.

File: draft.md [hunk 3]
```diff
@@ -129,45 +129,45 @@ The native Emotiq token (EMTQ) has deflationary economics, with the token supply
...
 
 # 5. Token Generation Event
 
-Emotiq plans to raise $60M (million) through the combination of a private sale and a public token generation event (TGE). The fundraising target has been carefully selected to enable us to reach our development milestones and jumpstart the Emotiq ecosystem. Emotiq will generate one billion EMTQ tokens, distributed as follows:
+Emotiq plans to raise $39M (million) through multiple private sale rounds. The fundraising target has been carefully selected to enable us to reach our development milestones and jumpstart the Emotiq ecosystem. Emotiq will generate one billion EMTQ tokens, distributed as follows:
 * 15% sold to seed contributors (completed Dec 2017);
-* 14% sold during the private sale;
-* 36% sold during the public TGE; and
-* 35% to be held by the Emotiq team, to be used as detailed later in this section.
+* 34% sold during the private sales;
```
Comment (joelreymont) [inline draft.md:138]: Can you double-check the numbers? We are selling 51% of the tokens to the public and the numbers do not add up to 49%.

File: draft.md [hunk 4]
```diff
@@ -129,45 +129,45 @@ The native Emotiq token (EMTQ) has deflationary economics, with the token supply
...
+* 10% to be held by the Emotiq team;
 
 Emotiq will follow strict KYC/AML procedures during both stages of fundraising, with funds used to, among other things:
 * Expand the the Emotiq ecosystem through educational programs;
 * Build a world-class in-house research team;
 * Explore the different implementations of smart contract languages and their usability, to accelerate blockchain adoption across the mass-market and enterprise; and
 * Provide venture funding for early projects on the Emotiq blockchain.
 
-Emotiq will aggressively pursue development, acquire the best talent, and explore new solutions. Its vision is ambitious, with equally ambitious goals.
+Emotiq will aggressively pursue development, acquire the best talent, and explore new solutions. The vision is ambitious, with equally ambitious goals.
```
Comment (joelreymont) [inline draft.md:148]: Let’s make it “Our vision is ambitions” or use “The Emotiq vision is ambition”. I don’t like the hanging “the” in “The vision”.

File: draft.md [hunk 5]
```diff
@@ -129,45 +129,45 @@ The native Emotiq token (EMTQ) has deflationary economics, with the token supply
...
 
-The private sale will raise $10M (million) from institutional and large private contributors and will follow a strict selection process. Single contributions will have a minimum of $500K (thousand) and contributors will be selected on the basis of the benefit they can bring to Emotiq and the ecosystem, e.g., promotion, publicity, exchange access, etc.
+The private sales will raise $37M (million) from institutional and large private contributors and will follow a strict selection process. Single contributions will have a minimum of $100K (thousand) and contributors will be selected on the basis of the benefit they can bring to Emotiq and the ecosystem, e.g., promotion, publicity, exchange access, etc.
```
Comment (joelreymont) [inline draft.md:150]: We are raising 39M and said so at the beginning. Why 37M here?

File: draft.md [hunk 6]
```diff
@@ -129,45 +129,45 @@ The native Emotiq token (EMTQ) has deflationary economics, with the token supply
...
-
-Combined, the private sale and public TGE are intended to raise $60M.
+Combined, the seed round and private sales are intended to raise $39M.
 
 * Team tokens will be locked up for 18 months.
 * Emotiq will set aside a token reserve which will be 100% used for promotion, publicity, airdrops, community expansion, bounties, etc. as determined by the Emotiq team.
-* There will be a 12-month lockup for seed contributors.
-* There will be a 6-month lockup for 50% of private sale contributor tokens.
-* Any tokens unsold after the private sale or public TGE will be kept in reserve by Emotiq, to be used for future funding needs.
+* There is a 12-month lockup for seed contributors.
```
Comment (joelreymont) [inline draft.md:158]: We should describe the exact lockup structure here. Shaishav has it.

File: draft.md [hunk 7]
```diff
@@ -241,62 +245,69 @@ Emotiq is private, scalable, and natural. Research-backed and supported by a tea
...
 China Community Manager
 
 Wen is the China Community Manager for Emotiq. With a deep knowledge of the Chinese cryptocurrency space and thorough understanding of blockchain projects, he provides the Emotiq team with expert regional advice, in addition to offering Chinese language support across all of Emotiq's social media channels.
 
+**Eric Yao**
+Writer and Editor
+
+Eric is the writer and editor for Emotiq. He has deep experience researching and writing about the cryptocurrency space and blockchain technology. He is excited to help create clear and effective PR, content, copy, and communications.
+
+**Ann Söderblom** Project Manager
```
Comment (joelreymont) [inline draft.md:295]: Let’s make it “Marketing” instead of “Project Manager”.

**Review by joelreymont — CHANGES_REQUESTED** [review]

File: draft.md (hunk 2 above)
Comment (cozybun) [reply draft.md:114]: Done!

File: draft.md (hunk 1 above)
Comment (cozybun) [reply draft.md:23]: Done!

File: draft.md (hunk 4 above)
Comment (cozybun) [reply draft.md:148]: Let's stick with "The Emotiq vision."

File: draft.md (hunk 5 above)
Comment (cozybun) [reply draft.md:150]: I subtracted the seed round to get to 37M because the total 39M is stated later. What do you think?

File: draft.md (hunk 7 above)
Comment (cozybun) [reply draft.md:295]: Done!

Comment (joelreymont) [reply draft.md:295]: Eric, you are making changes to a PR that has been merged already! Stop!!!

**Pull request merged**